test/e2e/pages/allocation-criteria.js

```diff
@@ -31,10 +36,20 @@ class AllocationCriteriaPage extends Page {
     await t.expect(this.getCurrentUrl()).contains(this.url)
 
     const hasOtherCriteriaAnswer = faker.random.arrayElement(['Yes', 'No'])
+    const estateAnswer = faker.random.arrayElement([
+      'Adult male',
+      'Adult female',
+      'Juvenile male',
+      'Juvenile female',
+      'Youth male',
+      'Youth female',
+      'Other',
+    ])
     const fieldsToFill = {
-      prisonerCategory: {
-        selector: this.fields.prisonerCategory,
+      estate: {
+        selector: this.fields.estate,
         type: 'radio',
+        value: estateAnswer,
```

Comment:
So you can probably skip needing this as I think if you don't pass a value to the field it will randomly select one of the options.

Reply:
Also, are these values reference data? If so, it's probably better to get the values from there, as they may change.